### PR TITLE
[Outreachy Task Submission] Added a success message once a collection is successfully added

### PR DESCRIPTION
**Pull request by kivuvarosekivuvan**
# Engineers Checklist

## What this PR does?
-Enhances the user experience by implementing a success message that confirms the successful addition and saving of a collection.

## How can it be tested?
- Launch the local server: http://localhost:4200
- Login/signup to the platform
- Navigate to the collection section on the sidebar
- Pop up modal opens up, click on "add collection" button
- Form is displayed, fill in the form and "Save" it

# Where is it documented?
- It is documented on an issue I raised: https://github.com/ushahidi/platform/issues/4826

## What would be the Impact of the solution?
- Implementing a success message upon adding and saving a collection would prevent users from creating duplicate collections and eliminate the need for manual confirmation of the collection list, improving user experience and increasing efficiency.

### The confirmation message once "Save" button is clicked

Group 9

[Screencast from 03-14-202
4 09:59:09 AM.webm](https://github.com/ushahidi/platform-client-mzima/assets/124133577/b08f79e8-85a9-49fc-b0ea-81ab13f876f1)

### CODE REVIEW

**Comment by kivuvarosekivuvan** [comment]
Hello @Angamanga and @ushahidlee, This is a solution I have developed from this [issue](https://github.com/ushahidi/platform/issues/4826) that I raised. I kindly ask for your review and feedback when you have a moment. Thanks

File: package.json [hunk 1]
```diff
@@ -115,7 +115,7 @@
     "jest-environment-jsdom": "29.4.3",
     "jest-preset-angular": "13.0.0",
     "lint-staged": "^13.1.2",
-    "nx": "16.0.2",
+    "nx": "^18.0.7",
     "nx-cloud": "16.0.5",
```
Comment (Angamanga) [inline package.json]: Remove this change for now, since its not affecting the actual issue.

File: package.json [hunk 2]
```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "@angular-material-components/datetime-picker": "^8.0.0",
     "@angular/animations": "~14.2.7",
-    "@angular/cdk": "^14.2.5",
+    "@angular/cdk": "^14.2.7",
     "@angular/common": "~14.2.7",
```
Comment (Angamanga) [inline package.json]: Remove this change for now, since its not affecting the actual issue.

**Review by Angamanga — CHANGES_REQUESTED** [review]
Very nice addition, I also saw this issue the other day! However, I think you can use the confirm-dialog and snackbar we already have in the client to solve this?

For the package-lock file, you could remove that from this PR, as well as the changing versions of some of the libraries. I generally try to avoid those kind of changes if they aren't absolutely needed for the specific change I am making :) 

Translations should go to Transifex instead, if you are interested in adding translations, please let me know :)

File: package.json (hunk 2 above)
Comment (kivuvarosekivuvan) [reply package.json]: Okay, I will work on these changes @Angamanga

**Review by Angamanga — APPROVED** [review]
Excellent! Here, you managed to revert the changes 🎉 Well done! Approved!

**Comment by kivuvarosekivuvan** [comment]
> Excellent! Here, you managed to revert the changes 🎉 Well done! Approved!

@Angamanga Thank you so much for approving this and at first giving me your thoughts on what to change, that's so good of you. Much appreciated

**Comment by Angamanga** [comment]
Hi @kivuvarosekivuvan! I changed base because I will make a small change in the text for the notification before merging it 👍

**Comment by kivuvarosekivuvan** [comment]
> Hi @kivuvarosekivuvan! I changed base because I will make a small change in the text for the notification before merging it 👍 

Hello Anna,

Thank you for this clarification.

**Pull request merged**